<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shop</class>
 <widget class="QMainWindow" name="Shop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shop</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 18px</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="styleSheet">
             <string notr="true">border: none;
text-align: center;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="styleSheet">
             <string notr="true">padding: 5px;
border-radius: 5px;
-webkit-appearance: none;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_2">
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: red;
border: none;
border-radius: 5px;
padding: 5px;</string>
            </property>
            <property name="text">
             <string>Очистить корзину</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="confirm">
            <property name="styleSheet">
             <string notr="true">border: none;
border-radius: 5px;
padding: 5px;
background: yellow;
color: black;</string>
            </property>
            <property name="text">
             <string>Добавить в корзину</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="confirmed">
            <property name="styleSheet">
             <string notr="true">background-color: green;
color: white;
border: none;
border-radius: 5px;
padding: 5px;</string>
            </property>
            <property name="text">
             <string>Подтвердить заказ</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="styleSheet">
           <string notr="true">border: none</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="main_request">
      <property name="styleSheet">
       <string notr="true">background: rgb(0, 102, 204);
color: white;
border: none;
border-radius: 5px;
padding: 5px;</string>
      </property>
      <property name="text">
       <string>Запросить товар с основного склада</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
